src/components/views/dialogs/spotlight/SpotlightDialog.tsx

```diff
@@ -523,18 +540,32 @@ const SpotlightDialog: React.FC<IProps> = ({ initialText = "", initialFilter = n
     if (trimmedQuery || filter !== null) {
         const resultMapper = (result: Result): JSX.Element => {
             if (isRoomResult(result)) {
+                const notification = RoomNotificationStateStore.instance.getRoomState(result.room);
+                const ariaProperties = {
+                    "aria-label": result.room.name,
+                    "aria-description": roomAriaDescription(result.room, notification),
```

Comment:
This is not ready for use, it is a W3C Editor's Draft

https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-description

Reply:
Although aria-description is just a draft it's already implemented in Chrome and Firefox.

Reply:
We also need Safari support though